pkg/formatters/emptyformatter.go

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright 2020 VMWare.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package formatters
+
+type EmptyFormatter struct{}
+
+func (f *EmptyFormatter) Write(obj interface{}) (err error) {
+	return nil
+}
```

Comment:
Looks like a warning asking for a new line at end of file

Reply:
I removed the empty interface with the next PR . 
I but I have added here new line